CMakeLists.txt

```diff
@@ -45,7 +45,12 @@ set( HEADER_FILES
 	${HEADER_FOLDER}/date/tz_private.h
 )
 
-add_library( tz ${HEADER_FILES} ${SOURCE_FOLDER}/tz.cpp )
+if (BUILD_SHARED_LIBS)
```

Comment:
no need for this if-else block, `add_library` recognizes `BUILD_SHARED_LIBS`

> If no type is given explicitly the type is STATIC or SHARED based on whether the current value of the variable BUILD_SHARED_LIBS is ON

https://cmake.org/cmake/help/v3.14/command/add_library.html?highlight=add_library

Reply:
Thanks - amended.